gc_registry/tests/measurement/test_measurement_routes.py

```diff
@@ -0,0 +1,37 @@
+
+
+from unittest.mock import MagicMock
+import pandas as pd
+import pytest
+
+
+@pytest.fixture
+def valid_measurement_json():
+
+    return """
+    [
+        {"device_id": "device_1", "interval_usage": 10, "interval_start_datetime": "2024-11-18T10:00:00", "interval_end_datetime": "2024-11-18T11:00:00"},
+        {"device_id": "device_1", "interval_usage": 15, "interval_start_datetime": "2024-11-18T11:00:00", "interval_end_datetime": "2024-11-18T12:00:00"},
+        {"device_id": "device_2", "interval_usage": 20, "interval_start_datetime": "2024-11-18T10:00:00", "interval_end_datetime": "2024-11-18T11:00:00"}
+    ]
+    """
+
+
+def test_submit_readings_success(api_client, db_write_session, db_read_session, esdb_client, valid_measurement_json, parsed_measurement_df, monkeypatch
```

Comment:
Had a go at adding a test, but it's complaining about the `measurement_json` being missing regardless of whether I use `json`, `data`, `params` arguments. Think maybe it needs to be in the request body